### PR TITLE
Create user-test-notes.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
This PR creates a markdown with all the notes from the Setember 2023 user testing sessions. It is still a WIP as there are some requested changes to make and a table of contents needs to be added.

### CODE REVIEW

**Comment by GeorgiaHCA** [comment]
Ready to review now!

File: 00-project-documentation/community/user-tests/user-test-notes.md [hunk 1]
```diff
@@ -0,0 +1,1238 @@
...
+## Demographic Information
+
+* No. Participants: 14 
+* Connections to autism: all autistic, 2 parents of autistic people, 3 with a professional connection to autism. 
+* Age ranges: 20s - 60s.   
+* Co-occuring conditions (across all participants): chronic health conditions, red/green colour blindness, agoraphobia, social anxiety, OCD, ADHD, hypomobility, EDS, partial deafness, childhood trauma, dyspraxia, depression, asthma, spacial disorientation, body dysmorphia, prior anorexia, asthma, hypervigilence 
+* Gender ratios: 1 non-binary, 9 female, 4 male
+* Socio-economic and educational backgrounds: range of education levels and professional backgrounds
+* Use of technology: all familiar with technology
+* Familiarity with AutSPACEs/citizen science: 4 out od 14 participants had already been involved in AutSPACEs, one had heard of it but not taken part and had been involved in similar research, the rest had no prior experience of citizen science or AutSPACEs
```
Comment (gedankenstuecke) [inline 00-project-documentation/community/user-tests/user-test-notes.md:12]:
```suggestion
* Familiarity with AutSPACEs/citizen science: 4 out of 14 participants had already been involved in AutSPACEs, one had heard of it but not taken part and had been involved in similar research, the rest had no prior experience of citizen science or AutSPACEs 
```

File: 00-project-documentation/community/user-tests/user-test-notes.md [hunk 2]
```diff
@@ -0,0 +1,1238 @@
...
+
+## Demographic Information
+
+* No. Participants: 14 
+* Connections to autism: all autistic, 2 parents of autistic people, 3 with a professional connection to autism. 
+* Age ranges: 20s - 60s.   
+* Co-occuring conditions (across all participants): chronic health conditions, red/green colour blindness, agoraphobia, social anxiety, OCD, ADHD, hypomobility, EDS, partial deafness, childhood trauma, dyspraxia, depression, asthma, spacial disorientation, body dysmorphia, prior anorexia, asthma, hypervigilence 
+* Gender ratios: 1 non-binary, 9 female, 4 male
+* Socio-economic and educational backgrounds: range of education levels and professional backgrounds
+* Use of technology: all familiar with technology
```
Comment (gedankenstuecke) [inline 00-project-documentation/community/user-tests/user-test-notes.md:11]:
```suggestion
* Use of technology: all familiar with technology, some actively engaged in own social media/online platform efforts, some more as consumers
```

File: 00-project-documentation/community/user-tests/user-test-notes.md [hunk 3]
```diff
@@ -0,0 +1,1238 @@
+# User Testing September 2023
+
```
Comment (gedankenstuecke) [inline 00-project-documentation/community/user-tests/user-test-notes.md:2]:
```suggestion
## Table of Contents
  * [Demographic Information](#demographic-information)
  * [User Test 1](#user-test-1)
  * [User Test 2](#user-test-2)
  * [User Test 3](#user-test-3)
  * [User Test 4](#user-test-4)
  * [User Test 5](#user-test-5)
  * [User Test 6](#user-test-6)
  * [User Test 7](#user-test-7)
  * [User Test 8](#user-test-8)
  * [User Test 9](#user-test-9)
  * [User Test 10](#user-test-10)
  * [User Test 11](#user-test-11)
  * [User Test 12](#user-test-12)
  * [User Test 13](#user-test-13)
  * [User Test 14](#user-test-14)
```

**Review by gedankenstuecke — APPROVED** [review]
This is really great, I've just made some suggested edits here and there so far!

**Comment by GeorgiaHCA** [comment]
Wonderful! Thanks @gedankenstuecke and thanks for doing the contents 🌟

**Pull request merged**